src/lib.rs

```diff
@@ -44,7 +44,7 @@
 //! 	.learn_momentum(0.6) // use the given learn momentum
 //! 	.log_config(LogConfig::Iterations(100)) // log state every 100 iterations
 //! 	.scheduling(Scheduling::Random)         // use random sample scheduling
-//! 	.criterion(Criterion::RecentMSE(0.03))  // train until the recent MSE is below 0.03
+//! 	.criterion(Criterion::RecentMSE(0.0001))  // train until the recent MSE is below 0.0001
```

Comment:
was this tested so that the tests wont take ages?

Reply:
Tested on my machine, and the CI. 

The alternative is switching this test off completely for now.

I don't think there is a good solution, apart from using a seeded random number generator. But I don't know how much work that would be.